asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js

```diff
@@ -91,6 +93,10 @@ const ExportPage = (props) => {
     setExporting(true);
   };
 
+  const disableRIS = () => {
+    return !props.dataWriter.includes("ris-writer");
+  };
```

Comment:
@J535D165 Available writers are returned in the API, but we still need an indicator to disable an option.